src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
The stylesheet link references a '.scss' file. Make sure this file is compiled to CSS and the correct path to the compiled CSS file is provided here. Browsers cannot directly interpret SCSS files.